docker-compose-clamav.yml

```diff
@@ -0,0 +1,10 @@
+version: '3.4'
```

Comment:
Where is this file used?

Reply:
It's for local development for a native or hybrid set up.